cli/flox-watchdog/src/logger.rs

```diff
@@ -68,3 +71,232 @@ pub(crate) async fn log_heartbeat() -> Sender<()> {
 
     stop_tx
 }
+
+/// Garbage collects known log files. Errors are traced and not bubbled up.
+pub(crate) async fn gc_logs(dir: impl AsRef<Path> + std::marker::Send + 'static) {
+    tokio::spawn(async move {
+        gc_logs_watchdog(&dir, KEEP_WATCHDOG_DAYS)
+            .unwrap_or_else(|err| error!(%err, "failed to delete watchdog logs"));
+        gc_logs_services(&dir, KEEP_SERVICES_LAST)
+            .unwrap_or_else(|err| error!(%err, "failed to delete services logs"));
+    });
+}
```

Comment:
suggestion: use sync threading

the spawned task does not perform any "async"/non-blocking work.
In fact gc_logs_* are both performing fs tasks using blocking io.
If we truly need tokio, this should then rather be spawned with `spawn_blocking`, although using plain threading should suffice here.

Reply:
Rebased into 26aafc6:

- `std::thread::spawn` instead of `tokio`
- renamed the function and update the comment to reflect changes
- redefines `dir` before moving into closure